ambari-server/src/main/java/org/apache/ambari/server/topology/TopologyManager.java

```diff
@@ -389,7 +376,7 @@ void saveOrUpdateQuickLinksProfile(String quickLinksProfileJson) {
   private void submitCredential(String clusterName, Credential credential) {
 
     ResourceProvider provider =
-        ambariContext.getClusterController().ensureResourceProvider(Resource.Type.Credential);
+        AmbariContext.getClusterController().ensureResourceProvider(Resource.Type.Credential);
```

Comment:
Would it be better here to use the method on AmbariContext to obtain this value from the "ambariContext" field in this class?

Reply:
Access to `static` members is generally preferred via the class, not instances (eg. IDEA produces warnings for the latter).